src/main/kotlin/com/soywiz/korge/intellij/editor/tiled/TiledMapWriter.kt

```diff
@@ -223,13 +315,13 @@ private fun XmlBuilder.propertiesToXml(properties: Map<String, TiledMap.Property
 	node("properties") {
 		properties.forEach { (name, prop) ->
 			when (prop) {
-				is TiledMap.Property.StringT -> property(name, "string", prop.value)
-				is TiledMap.Property.IntT -> property(name, "int", prop.value)
-				is TiledMap.Property.FloatT -> property(name, "float", prop.value)
-				is TiledMap.Property.BoolT -> property(name, "bool", prop.value.toString())
-				is TiledMap.Property.ColorT -> property(name, "color", prop.value.toStringARGB())
-				is TiledMap.Property.FileT -> property(name, "file", prop.path)
-				is TiledMap.Property.ObjectT -> property(name, "object", prop.id)
+				is Property.StringT -> property(name, "string", prop.value)
```

Comment:
Even when in packages I usually like to either limit the scope of the class by putting in inside other class like TiledMap.Property, or to prefix with TiledMapProperty. The reason is that with global autocompletion there might be several unrelated Property classes that might be confusing to people or annoying when importing the wrong one

Reply:
Well, here the import for `TiledMap.*` was added, and IDEA suggested remove redundant notices

Reply:
then it's okay, good